.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Description
+A few sentences describing the overall goals of the issue.
+
+## Depends On
+ - A list of other involved issues (with link)
+ - ...
+
+**Please indicate if this issue is related to a bug or a new feature request**
+- [ ] Bug
```

Comment:
We are already using labels for this.

Reply:
The manager will associate label properly

Reply:
In general. why we repeat the same information in many different places?